src/main/Combatant.ts

```diff
@@ -107,13 +109,30 @@ export default class Combatant {
     this._realm = value;
   }
 
+  /**
+   * Gets the region.
+   */
+  get region() {
+    return this._region;
+  }
+
+  /**
+   * Sets the region.
+   */
+  set region(value) {
+    this._region = value;
+  }
+
   isFullyDefined() {
     const hasGUID = this.teamID !== undefined;
     const hasName = this.name !== undefined;
     const hasRealm = this.realm !== undefined;
+    const hasRegion = this.region !== undefined;
     const hasSpecID = this.specID !== undefined;
     const hasTeamID = this.teamID !== undefined;
-    return hasGUID && hasName && hasRealm && hasSpecID && hasTeamID;
+    return (
+      hasGUID && hasName && hasRealm && hasRegion && hasSpecID && hasTeamID
```

Comment:
I think `isFullyDefined` should probably not check the region given it may be undefined. I think this fn is used to reduce log parsing overhead particularly in classic or game modes where there is no COMBATANT_INFO.